File: Coding/03_Beginner/Full_search/JavaScript/7.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split(/\s+/);

const N = Number(input[0]);
const A = input.slice(1).map(Number);

const maxA = Math.max(...A);
let cnt = 0;

for (let i = 0; i < N; i++){
    if (A[i] == maxA){
        cnt++;
    }
}

console.log(cnt);

File: Coding/03_Beginner/set_dict/Python/7.py
N = int(input())
A = list(map(int, input().split()))

d = {}
for a in A:
    d[a] = d.get(a, 0) + 1

ans = 0
for v, c in d.items():
    if c >= v:
        ans += c - v
    else:
        ans += c

print(ans)
